test/integration/scheduler/volume_binding_test.go

```diff
@@ -416,9 +428,26 @@ func setupCluster(t *testing.T, nsName string, numberOfNodes int) *testConfig {
 			clientset.CoreV1().PersistentVolumes().DeleteCollection(nil, metav1.ListOptions{})
 			clientset.StorageV1().StorageClasses().DeleteCollection(nil, metav1.ListOptions{})
 			cleanupTest(t, context)
-			utilfeature.DefaultFeatureGate.Set("VolumeScheduling=false,LocalPersistentVolumes=false")
 		},
 	}
+
+	if len(featureStr) != 0 {
+		config.teardown = func() {
```

Comment:
An explicit option for doing teardown may be better than the presence of a feature string.

Reply:
Sorry I didn't quite follow here :D 

What kind of refactoring you are suggesting in order to let `setupCluster` enable feature gates on demand?

Reply:
Instead of declaring 2 different functions for teardown based on featureStr, can you just check for the featureStr at L442?